### PR TITLE
#235 Use a global variable instead of a same variable in each file

### PR DESCRIPTION
**Pull request by fkroockmann**
Change #bb5-ui to use a Core.config

### CODE REVIEW

File: src/tb/apps/bundle/views/bundle.view.list.js [hunk 1]
```diff
@@ -16,149 +16,164 @@
...
+                return this;
+            }
+        });
 
-    return BundleViewList;
-});
\ No newline at end of file
+        return BundleViewList;
+    }
+);
```
Comment (ndufreche) [inline src/tb/apps/bundle/views/bundle.view.list.js]: I don't understand why there is so much changes in this file.

Comment (fkroockmann) [inline src/tb/apps/bundle/views/bundle.view.list.js]: Cause i reorganize a define

Comment (ndufreche) [inline src/tb/apps/bundle/views/bundle.view.list.js]: ok

**Comment by ndufreche** [comment]
:+1:

**Pull request merged**